README.rst

```diff
@@ -0,0 +1,158 @@
+========================================================
+ The Community Earth System Model version 2.0 (CESM2.0)
+========================================================
+
+See the CESM web site for documentation and information:
+http://www2.cesm.ucar.edu
+
+.. sectnum::
+
+.. contents::
+
+Obtaining the full model code and associated scripting infrastructure
+=====================================================================
+
+CESM2.0 is now released via github. You will need some familiarity with git in order
+to modify the code and commit these changes. However, to simply checkout and run the
+code, no git knowledge is required other than what is documented in the following steps.
+
+To obtain the CESM2.0 code you need to do the following:
+
+#. Clone the repository. ::
+
+      > git clone https://github.com/escomp/cesm.git my_cesm_sandbox
+
+   This will create a directory ``my_cesm_sandbox/`` in your current working directory.
+
+#. Go into the newly created CESM repository and determine what version of CESM you want.
+   To see what cesm tags are available, simply issue the **git tag** command. ::
+
+      > cd my_cesm_sandbox
+      > git tag
+
+#. Do a git checkout of the tag you want. If you want to checkout cesm2.0.beta07, you would issue the following. ::
+
+      > git checkout cesm2.0.beta07
+
+#. Run the script **manage_externals/checkout_externals.py**. ::
+
+      > ./manage_externals/checkout_externals.py
+
+   The **checkout_externals.py** script is a package manager that will
+   populate the cesm directory with the relevant versions of each of the
+   components along with the CIME infrastructure code.
+
+At this point you have a working version of CESM.
+
+To see full details of how to set up a case, compile and run, see the CIME documentation at http://esmci.github.io/cime/ .
+
+More details on checkout_externals.py
+-------------------------------------
+
+The file **CESM.cfg** in your top-level CESM directory tells
+**checkout_externals.py** which tag/branch of each component should be
+brought in to generate your sandbox. (This file serves the same purpose
+as SVN_EXTERNAL_DIRECTORIES when CESM was in a subversion repository.)
+
+**You need to rerun checkout_externals.py whenever CESM.cfg has
+changed** (unless you have already manually updated the relevant
+external(s) to have the correct branch/tag checked out). Common times
+when this is needed are:
+
+* After checking out a new CESM branch/tag
+
+* After merging some other CESM branch/tag into your currently
+  checked-out branch
+
+**checkout_externals.py** must be run from the root of the source tree -
```

Comment:
There is subtlety to the definition of 'root of the source tree' that confused Alice. I think you want to refer to only running it from cesm, not components. See:

https://github.com/NCAR/manage_externals/blob/bb0f5a04d458925c3b82a4cb680ec7014b99e339/checkout_externals.py#L56

Reply:
Good point. Reworded version incoming.